.gitignore

```diff
@@ -3,3 +3,8 @@
 **/install
 **/logs
 .vscode
+DroneCore.config
```

Comment:
Can you make a comment where these are from?

Reply:
ok, I'll add it. It comes from Qt  Creator